src/test/regress/expected/multi_follower_select_statements.out

```diff
@@ -159,6 +159,29 @@ SELECT * FROM the_table;
  1 | 2
 (2 rows)
 
+-- Check for connectivity in the cluster
+-- verify that we check connections to followers as well.
```

Comment:
Ops, this looks very unusual output? Shouldn't we have 4 rows?  Is it leftover from the previous nodeList in the code? It'd be nice to do `SELECT * FROM citus_check_cluster_node_health()`

Reply:
This is a follower test, and in this test setup we have followers in the cluster.

2 primary workers + 2 followers = 4
4 x 4 = 16

Let me use your suggested query and update the comments.